src/jmcomic/jm_async_client.py

```
@@ -0,0 +1,737 @@
+    async def req_api(self,
+                      url: str,
+                      get: bool = True,
+                      require_success: bool = True,
+                      params: dict | None = None,
+                      **kwargs,
+                      ) -> JmApiResp:
+        """
+        核心的 API 请求封装方法。
+        处理参数拼装、请求发送与重试，并返回统一的 JmApiResp 响应对象。
+        """
+        # /setting 是 setup() 内部初始化调用的接口，跳过 setup 防止 asyncio.Lock 不可重入死锁
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)